app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,9 @@
   </head>
 
   <body>
+  <% flash.each do |type, message| %>
+      <p><%= message %></p>
+    <% end %>
```

Comment:
Great use of this! This will allow others to refactor their code later since they put their flash statement in the direct index file.